aws/modules/consolidate-logs/module.ftl

```diff
@@ -0,0 +1,226 @@
+                                    }
+                                },
+                                "LogWatchers": {
+                                    "app-all": {
+                                        "LogFilter": "all-logs"
+                                    }
+                                },
+                                "aws:WAFLogFeed": true,
+                                "Links": {
+                                    "store": {
+                                        "Tier": "mgmt",
+                                        "Component": "baseline",
+                                        "DataBucket": "opsdata",
+                                        "Instance": "",
+                                        "Version": ""
+                                    },
+                                    "processor" : {
+                                        "Tier" : tier,
+                                        "Component" : lambdaName,
+                                        "Instance" : "",
+                                        "Version" : "",
+                                        "Function" : "processor",
+                                        "Role" : "invoke"
+                                    }
+                                }
+                            }
+                        },
+                        lambdaName : {
+                            "lambda": {
+                                "Instances": {
+                                    "default": {
+                                        "DeploymentUnits": [ lambdaName ]
+                                    }
+                                },
+                                "Image" : {
+                                    "Source" : "url",
+                                    "UrlSource" : {
+                                        "Url" : lambdaSourceUrl,
+                                        "ImageHash" : lambdaSourceHash
+                                    }
+                                },
+                                "Functions": {
+                                    "processor": {
+                                        "RunTime": "python3.6",
+                                        "MemorySize": 128,
+                                        "Timeout": 30,
+                                        "Handler": "src/run.lambda_handler",
+                                        "Links": {
+                                            "store": {
+                                                "Tier": "mgmt",
+                                                "Component": "baseline",
+                                                "DataBucket" : "opsdata",
+                                                "Instance" : "",
+                                                "Version": "",
+                                                "Role": "datafeed"
+                                            }
```

Comment:
I don't think the lambda function has any knowlege of the S3 bucket only the firehose should have that data

Reply:
Cheers, removed.